recipes/mutpy-pynguin/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mutpy-pynguin" %}
+{% set version = "0.7.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/MutPy-Pynguin-{{ version }}.tar.gz
+  sha256: 31fe168eff221ece0129768b68375ca0d03c6514567caf19878ae7a618b9ab89
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - mutpy ==0.6.1 pyhd8ed1ab_4
+    - pip
+    - python >=3.8
+  run:
+    - astmonkey >=0.3.6
+    - jinja2 >=2.7.1
+    - mutpy ==0.6.1 pyhd8ed1ab_4
```

Comment:
Upstream says this is a fork of `mutpy`...so why are you depending on it? I don't see it in the upstream requirements.

Reply:
to be honest, it's been so long I don't remember. Let me try taking it ou.

Reply:
looks like it's working? I really don't know/remember why I was trying to ad mutpy as a dependency. Thanks for catching this!